apps/stripe/src/app/api/saleor/transaction-initialize-session/use-case.ts

```diff
@@ -0,0 +1,102 @@
+import { buildSyncWebhookResponsePayload } from "@saleor/app-sdk/handlers/shared";
+import { err, ok } from "neverthrow";
+
+import { TransactionInitializeSessionEventFragment } from "@/generated/graphql";
+import { BaseError } from "@/lib/errors";
+import { createLogger } from "@/lib/logger";
+import { AppConfigPersistor } from "@/modules/app-config/app-config-persistor";
+import { SaleorApiUrl } from "@/modules/saleor/saleor-api-url";
+import { IStripePaymentIntentsApiFactory } from "@/modules/stripe/types";
+
+export class InitializeStripeTransactionUseCase {
```

Comment:
Maybe let's bind use cases names to routes/mutations? We agreed we will have 1:1 mapping so it would be good convention